odl/space/fspace.py

```diff
@@ -372,8 +372,8 @@ def __call__(self, x, out=None, **kwargs):
 
         # Call the function and check out shape, before or after
         if out is None:
-            try:
-                if ndim == 1:
+            if ndim == 1:
+                try:
```

Comment:
This was mostly done since something else had I bug I think, so there is a bug-fix involved here as well, I think it had something to do with the function getting called twice in the case of 1d problem.